README.md

```diff
@@ -22,7 +22,8 @@ __Windows__
 * [Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx)
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
-* [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Python 2.7+ Or 3.4+](https://www.python.org/downloads/) (add to PATH)
```

Comment:
@Zangetsu38 I doesn't matter if it's old or "no one" uses it. (I would bet a huge chunk of Python 3.* installations are below 3.4) Requirements are there to state, what version **works**, not what version we recommend you to use because it's "too old". Python 2.7.0 was first released 5 years ago, so by your logic, we shouldn't list it as compatible and simply assume not list it as compatible (while it very very likely is), because it's too old.